Cargo.toml

```diff
@@ -40,6 +39,7 @@ indoc = { version = "2.0" }
 regex = { version = "1.11.1" }
 uuid = { version = "1.10", features = ["v3", "v4", "serde"] }
 dyn-clone = { version = "1.0" }
+once_cell = { version = "1.20.2" }
```

Comment:
- I initially chose `once_cell` because `std::cell::OnceCell` is not thread-safe, which resulted in the following error.  
- I have now switched to `std::sync::OnceLock`, which is thread-safe and part of the standard library, thereby eliminating the dependency on the `once_cell` crate.

```shell
error[E0277]: `std::cell::OnceCell<sqlx::Pool<sqlx::Postgres>>` cannot be shared between threads safely
   --> swiftide-integrations/src/pgvector/persist.rs:13:18
    |
13  | impl Persist for PgVector {
    |                  ^^^^^^^^ `std::cell::OnceCell<sqlx::Pool<sqlx::Postgres>>` cannot be shared between threads safely
    |
    = help: the trait `std::marker::Sync` is not implemented for `std::cell::OnceCell<sqlx::Pool<sqlx::Postgres>>`, which is required by `pgvector::PgVector: std::marker::Sync`
```